### PR TITLE
kash's pull request

### PR DESCRIPTION
**Pull request by kashifameen**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+	
+}
+.instagram{
+	margin: 10px 10px;
+}
+.instagram:hover{
+	border: 1px solid #fff;
+	border-radius: 8px;
+	transform: scale(1.4);
+}
```
Comment (dotNATE) [inline style.css:318]: The icons from 282-318 all have v similar scale transformations on :hover. Could be refactored to save lines of code.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+	
+}
+
+.contact button:hover{
+	cursor: pointer;
+	transform: scale(1.2);
+	color: #A32CC4;
+}
+
+
```
Comment (isabellecochrane) [inline style.css:202]: lots of line breaks

File: style.css [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+	background: black;
+	color: white;
+	justify-content: center;
+	
+}
+.contactform form {
+	background: transparent;
+}
+
+label { display:block }
```
Comment (isabellecochrane) [inline style.css:223]: need to reformat this one

File: style.css [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+.contactform input{
+	width: 100%;
+	background: transparent;
+	color: #fff;
+	padding-top: 5px;
+	
+	
+}
+
+
```
Comment (isabellecochrane) [inline style.css:239]: lots of line breaks

File: style.css [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+	
+	.contact button:hover{
+		transform: scale(1.0);
+	}
+}
+
+@media only screen and (max-width: 300px){
+	
+		
+}
```
Comment (dotNATE) [inline style.css:393]: empty media query

File: style.css [hunk 6]
```diff
@@ -0,0 +1,393 @@
...
+
+.contactform button:hover{
+	transform:scale(1.2);
+	margin-top: 10px;
+	color:#A32CC4;
+}
+
+footer {
+	padding-top: 50px;
+
```
Comment (isabellecochrane) [inline style.css:271]: assuming your probs gonna fill this?

File: style.css [hunk 7]
```diff
@@ -0,0 +1,393 @@
...
+	padding-top: 50px;
+	
+	
+}
+
+
+footer img{
+	height: 40px;
+	width: 40px;
+
```
Comment (isabellecochrane) [inline style.css:279]: same here

File: style.css [hunk 8]
```diff
@@ -0,0 +1,393 @@
...
+.instagram:hover{
+	border: 1px solid #fff;
+	border-radius: 8px;
+	transform: scale(1.4);
+}
+
+@media only screen and (max-width:600px){
+	
+	
+	.homepage{
```
Comment (isabellecochrane) [inline style.css:323]: some have a space before { some dont

File: style.css [hunk 9]
```diff
@@ -0,0 +1,393 @@
...
+	border-radius: 40px;
+	padding-left: 20px;
+	text-align: center;
+	margin-bottom: 50px;
+	}
+	.skills .css{
+		border-left: none;
+	}
+	.container{
+			grid-template-rows: 1fr 1fr 1fr 1fr ;
```
Comment (isabellecochrane) [inline style.css:343]: these don't need to be so indented

**Review by dotNATE — COMMENTED** [review]
Could maybe do with adding comments to CSS to separate into sections. Everything reads well but I wouldn't be able to find anything specific if you asked me to!

The page looks great and seems to work well.

**Pull request closed without merging**